api/handler/catalog.go

```diff
@@ -47,3 +51,12 @@ func (h *CatalogHandler) CreateCatalog(ctx echo.Context) error {
 		Id: id.(string),
 	})
 }
+
+func (h *CatalogHandler) ListCatalog(ctx echo.Context) error {
```

Comment:
Can you rename this to GetCatalog. We usually use the List methods when we want to return multiple resources (as in a Catalog list).